HHS/Head-Start-TTADP
# [PROD] Fix bug preventing removal of attachments

## Description of change
A bug was found where Objective files weren't cleaned up in the same way objective resources and topics were. 

If a user adds a file to a newly created objective on an activity report and then attempts to delete that file, an error on the 
backend will cause the objective removal to fail and desynchronize the frontend and the backend, making attempts to save the report to fail.

This PR fixes that bug.

## How to test
Create a report on staging with an objective and an attachment. Add a second objective, delete the first objective. Everything should be as normal when you save the goals and objectives section.


## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1500

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
